Greenheart/swipeforfuture.com
# feat: add world state extensions

This branch will:
* Add support for world state extensions
* Add rounds world state extension to default data
* Add cyclic hourOfDay world state extension to default data

This branch should not change the behaviour of any other state related evaluation

To test:
* Run application and monitor gameworld state
* Look for rounds that count up from 0
* Look for hourOfDay that should have a cycle from 0 to 23
* Verify that other state changes behaves as expected
